compiler/aarch64/codegen/OMRCodeGenerator.cpp

```diff
@@ -181,6 +181,11 @@ OMR::ARM64::CodeGenerator::initialize()
          {
          cg->setSupportsArrayCmp();
          }
+      static const bool disableArrayCmpLen = feGetEnv("TR_aarch64DisableArrayCmpLen") != NULL;
```

Comment:
Is there specific reason for using `TR_aarch64DisableArrayCmpLen` ?? i.e, why `aarch64` in the flag?

Reply:
This is following the pattern for setting the `supportsArrayCmp` flag [a few lines earlier](https://github.com/eclipse/omr/blob/104d1f60f5a242078a88364fe5b78f54ef544db2/compiler/aarch64/codegen/OMRCodeGenerator.cpp#L179)